crates/service/src/tap/receipt_store.rs

```diff
@@ -81,9 +165,7 @@ impl IndexerTapContext {
                 tokio::select! {
                     biased;
                     _ = receiver.recv_many(&mut buffer, BUFFER_SIZE) => {
-                        if let Err(e) = inner_context.store_receipts(buffer).await {
-                            tracing::error!("Failed to store receipts: {}", e);
-                        }
```

Comment:
I know what I'm suggesting above means bringing this back in, but it's easier to read (and more adaptable) to have an error return here.

Reply:
done